extensions/wikia/Email/Controller/ForgotPasswordController.class.php

```diff
@@ -14,75 +14,52 @@
  * @package Email\Controller
```

Comment:
Hello, and welcome to another installment of "Review the long email change set."  I'll be your host, @garthwebb, and today we'll be looking at a real classic.